src/omero/plugins/duplicate.py

```diff
@@ -51,7 +52,23 @@
     # that would have been duplicated
     omero duplicate Dataset:53 --dry-run --report
 
+    # Duplicate a project with its datasets but not their images
+    omero duplicate Project:15 --ignore=DatasetImageLink
+    # Duplicate a project with the original images linked from its datasets
+    omero duplicate Project:15 --reference=Image
 """
+        "    # Duplicate a project, linking to the original annotations "
```

Comment:
Is there a reason for the extra `"""` and separately quoted lines here?

Reply:
Minor point - shouldn't we use `Project` instead of `project` to refer to objects? Also it's more consistent with the commands themselves.

Reply:
Stops flake8 complaining about the long line.

Reply:
I'm trying to be consistent with the other CLI graphs command help text, this PR was originally just adapted from `GraphControl` methods which is why some of the suggested changes here I've mirrored back into omero-py. Certainly open to reworking it all but perhaps could be a separate effort given how many related subcommands there are.